docs/user/registering-microshift-devices-acm.md

```diff
@@ -0,0 +1,157 @@
+# Auto-Registering Devices with MicroShift into ACM
+
+## Configuring RHACM Agent Registration
```

Comment:
In general, let's spell out acronyms once "Advanced Cluster Management (ACM)" before using. In the heading, this would be too much, of course.

Reply:
+1